lib/mix/tasks/bench.ex

```diff
@@ -118,6 +123,19 @@ defmodule Mix.Tasks.Bench do
     end
   end
 
+  defp normalize_files({[file], opts}) do
+    case Regex.run(~r/^(.+):(\d+)$/, file, capture: :all_but_first) do
+      [file, line_number] ->
+        {[file], Keyword.put(opts, :line, String.to_integer(line_number))}
+      _ ->
+        {[file], opts}
+    end
+  end
+
+  defp normalize_files({paths, opts}) do
```

Comment:
What if multiple files with line numbers are passed on the command line? In that case, it will explode with a file not found error. We should either support running multiple benchmarks on specific lines or provide a more helpful error message. Up to you.